tests/e2e/support/mod.rs

```diff
@@ -3,6 +3,7 @@ pub mod client_capabilities;
 pub mod cursor;
 pub mod fixture;
 pub mod jsonrpc;
+pub mod load_data;
```

Comment:
either

```suggestion
pub mod data;
```

or maybe

```suggestion
pub mod insta;
```

?